subgraphs/_reference_/src/prices/config/gnosis.ts

```diff
@@ -0,0 +1,122 @@
+
+export const CURVE_REGISTRY_ADDRESSES: Address[] = [
+  Address.fromString("0x55E91365697EB8032F98290601847296eC847210"),
+  Address.fromString("0x8A4694401bE8F8FCCbC542a3219aF1591f87CE17"),
+];
+
+///////////////////////////////////////////////////////////////////////////
+/////////////////////////// UNISWAP FORKS CONTRACT ////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const UNISWAP_FORKS_ROUTER_ADDRESSES: Address[] = [
+  Address.fromString("0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506"), // SushiSwap
+];
+
+///////////////////////////////////////////////////////////////////////////
+/////////////////////////// BLACKLISTED TOKENS ////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const YEARN_LENS_BLACKLIST: Address[] = [];
+export const AAVE_ORACLE_BLACKLIST: Address[] = [];
+export const CURVE_CALCULATIONS_BLACKSLIST: Address[] = [];
+export const SUSHI_CALCULATIONS_BLACKSLIST: Address[] = [];
+
+///////////////////////////////////////////////////////////////////////////
+//////////////////////////// HARDCODED STABLES ////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const HARDCODED_STABLES: Address[] = [];
+
+///////////////////////////////////////////////////////////////////////////
+///////////////////////////////// HELPERS /////////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const USDC_TOKEN_DECIMALS = BigInt.fromI32(6);
+
+export const ETH_ADDRESS = Address.fromString(
+  "0x6a023ccd1ff6f2045c3309768ead9e68f978f6e1"
+);
+export const WETH_ADDRESS = Address.fromString(
+  "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
+);
+export const USDC_ADDRESS = Address.fromString(
+  "0xddafbb505ad214d7b80b1f830fccc89b60fb7a83"
+);
+
+export class config implements Configurations {
+  yearnLens(): Address {
+    return YEARN_LENS_CONTRACT_ADDRESS;
+  }
+  chainLink(): Address {
+    return CHAIN_LINK_CONTRACT_ADDRESS;
+  }
+  yearnLensBlacklist(): Address[] {
+    return YEARN_LENS_BLACKLIST;
+  }
+
+  aaveOracle(): Address {
+    return AAVE_ORACLE_CONTRACT_ADDRESS;
+  }
+  aaveOracleBlacklist(): Address[] {
+    return AAVE_ORACLE_BLACKLIST;
+  }
+
+  curveCalculations(): Address {
+    return CURVE_CALCULATIONS_ADDRESS;
+  }
+  curveCalculationsBlacklist(): Address[] {
+    return CURVE_CALCULATIONS_BLACKSLIST;
+  }
+
+  sushiCalculations(): Address {
+    return SUSHISWAP_CALCULATIONS_ADDRESS;
+  }
+  sushiCalculationsBlacklist(): Address[] {
+    return SUSHI_CALCULATIONS_BLACKSLIST;
+  }
+
+  uniswapForks(): Address[] {
+    return UNISWAP_FORKS_ROUTER_ADDRESSES;
+  }
+  curveRegistry(): Address[] {
+    return CURVE_REGISTRY_ADDRESSES;
+  }
+
+  hardcodedStables(): Address[] {
+    return HARDCODED_STABLES;
+  }
+
+  ethAddress(): Address {
```

Comment:
you could consider naming this networkTokenAddress or nativeAddress

Reply:
`ethAddress` and `wethAddress` parameters are named according to uniswap contract parameters. As all the uniswap forks contracts follow the same nomenclature, we should do the same.